src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesNotShown.jsx

```diff
@@ -34,8 +36,7 @@ export default function FacilitiesNotShown({
 
   const nearbyUnsupportedFacilities = facilities?.filter(
     facility =>
-      !facility.legacyVAR.settings[typeOfCareId]?.direct.enabled &&
```

Comment:
Moved this check into a service function, so I could add the Cerner check in one spot